frontend/src/scenes/cohorts/cohortEditLogic.ts

```diff
@@ -205,6 +206,7 @@ export const cohortEditLogic = kea<cohortEditLogicType>([
                 },
             }),
             submit: (cohort) => {
+                actions.submitCohort()
```

Comment:
this action exists by default thanks to forms, you shouldn't need to add it yourself, was this not working if you remove the action and this line?

[you can check the generated `cohortEditLogicType.ts` file for all existing actions

Reply:
Yeah, will check the functionality after removing this line. Will update the PR accordingly if it works

Reply:
this and the line in actions for the same 👍

Reply:
@neilkakkar, I have made the required changes. Can you please take a look!